package.json

```diff
@@ -84,12 +82,16 @@
     "stylelint": "^13.2.1",
     "stylelint-config-nebenan": "^1.6.0",
     "terser": "^4.6.7",
+    "nebenan-react-datepicker": "^1.0.0",
```

Comment:
nebenan-react-datepicker needs to be a direct dependency

nebenan-helpers needs to be a peer (loaded by multiple packages, need to insure single version)